services/workflows-service/src/alert/alert.service.ts

```diff
@@ -20,6 +22,32 @@ export class AlertService {
     return await this.prisma.alertDefinition.create({ data: dto as any });
   }
 
+  async updateAlertsAssignee(
+    alertIds: string[],
+    projectId: string,
+    assigneeDto: AlertAssigneeUniqueDto,
+  ): Promise<Alert[]> {
+    try {
+      return await this.alertRepository.updateMany(alertIds, projectId, {
+        data: {
+          assigneeId: assigneeDto.assigneeId,
+        },
+      });
+    } catch (error: unknown) {
+      // Should be handled by ProjectAssigneeGuard on controller level
+      if (error instanceof Prisma.PrismaClientKnownRequestError) {
```

Comment:
Replace with a method similar to `isRecordNotFoundError` found in `prisma.util.ts`.

Reply:
done